### ISSUE TITLE
Add installation instructions for LDNode

### ISSUE THREAD

**Issue by dmitrizagidulin** [solid/solid-signup#10]
I haven't been able to install this on LDNode. I suspect I'm running into some combination of:
- not having the right `.acl` file in the `solid-signup/` directory
- not being clear where the solid-signup/ app folder is supposed to be in relation to the `--root` parameter.

**Comment by nicola** [comment]
`--root` decides on the folder where you want to start ldnode from, let's say `--root myldp`, now in `myldp` you can drag and drop the folder (or git clone `solid-signup`) in this folder.

Then, you will have to set the ACL for this file to be public/readable (either manually or with a tool e.g. warp). I know it is very complex right now. We should maybe make an app that makes you install apps

**Comment by dmitrizagidulin** [comment]
Hmm, I thought I did both of those steps. Maybe I got the ACL wrong somehow..

**Comment by nicola** [comment]
Sure let me know! (still I think we need a sort of app that simplify app installation.. maybe this can be the default app installed?)

**Comment by dmitrizagidulin** [comment]
Yeah, we definitely do need to simplify it. And yes, that should definitely be one of the default apps!
